CMakeLists.txt

```diff
@@ -336,6 +336,9 @@ if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_B
     else()
         message(STATUS "Build without rocblas")
     endif()
+
+    # hipblaslt
+    find_package(hipblaslt REQUIRED PATHS /opt/rocm)
```

Comment:
Please make it `find_package(hipblaslt REQUIRED PATHS /opt/rocm $ENV{HIP_PATH})` to accommodate Windows.